src/components/dashboard/Claim.js

```diff
@@ -450,8 +450,8 @@ const getStylesFromProps = ({ theme }) => {
       display: 'flex',
       flexDirection: 'column',
       alignItems: 'center',
-      marginBottom: getDesignRelativeHeight(isSmallDevice ? 16 : 20),
-      marginTop: getDesignRelativeHeight(isBrowser ? 70 : 18),
+      marginBottom: getDesignRelativeHeight(isLargeDevice ? 20 : 16),
```

Comment:
explain why it all changed to isLargeDevice?

Reply:
The olds conditional don't look good on device like iPhone 8, this new conditional seems to look good on all devices.

Reply:
ok. 
@mauriguz @MathiasMoralesDixtra 
lets agree that during your local tests you will take screen snapshots of the different devices you test it on
specifically edge cases like small devices iphone5